src/index.ts

```diff
@@ -1,7 +1,7 @@
 import fs from "fs";
 import mkdirp from "mkdirp";
 import yargs, { Argv } from "yargs";
-import { obfuscateDocument, verifySignature } from "@govtechsg/open-attestation";
+import { obfuscateDocument, verifySignature, getData } from "@govtechsg/open-attestation";
```

Comment:
eslint will complain ;) (or should)

Reply:
Yes. i wanted to fix for issue #5 at the same time but i will wait until this feature is completed